app/models/user.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const logger = require('../logger');
+
+// User Model, including validation and corresponding error messages.
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('Users', {
+    firstName: {
+      type: DataTypes.TEXT,
+      allowNull: {
+        args: false,
+        msg: 'The first name field cannot be empty, please try again.'
+      },
+      notEmpty: {
+        msg: 'The first name field cannot be empty, please try again.'
+      }
+    },
+    lastName: {
+      type: DataTypes.TEXT,
+      allowNull: {
+        args: false,
+        msg: 'YThe last name field cannot be empty, please try again.'
```

Comment:
Wrong message here, maybe you can use a function for create all this messages something like: `emptyValidation = field => 'The ${field} field cannot be empty' `